model/src/wav_history_mod.F90

```diff
@@ -372,8 +437,9 @@ subroutine write_history ( timen )
         if(vname .eq.      'PNR') call write_var2d(vname, pnr      (1:nseal_cpl) )
 
         ! Group 5
-        if (vname .eq.    'USTX') call write_var2d(vname, ust      (1:nseal_cpl)*asf(1:nseal_cpl), dir=cos(ustdir(1:nseal_cpl)), usemask='true')
-        if (vname .eq.    'USTY') call write_var2d(vname, ust      (1:nseal_cpl)*asf(1:nseal_cpl), dir=sin(ustdir(1:nseal_cpl)), usemask='true')
+        if (vname .eq.    'USTX') call write_var2d(vname, ust      (1:nsea)*asf(1:nsea), dir=cos(ustdir(1:nsea)), init0='false', global='true')
+        if (vname .eq.    'USTY') call write_var2d(vname, ust      (1:nsea)*asf(1:nsea), dir=sin(ustdir(1:nsea)), init0='false', global='true')
+
```

Comment:
Have you run in debug mode and verified all the changed variables (ustx,usty, ths,psi) work as expected?